contracts/implementations/MultipleMinter.sol

```diff
@@ -164,7 +165,6 @@ contract MultipleMinter is
     function mintVehicleAndSdWithDeviceDefinitionSign(
         MintVehicleAndSdWithDdInput calldata data
     ) external onlyRole(MINT_VEHICLE_SD_ROLE) {
-        NodesStorage.Storage storage ns = NodesStorage.getStorage();
```

Comment:
Interesting—does this help?

Reply:
Yep... makes the code ugly but it does help to unload the stack.